services/core-web/src/components/common/CoreTableCommonColumns.tsx

```diff
@@ -93,16 +125,21 @@ export const renderDocumentLinkColumn = (
     dataIndex,
     key: dataIndex,
     render: (text = "", record: any) => {
+      const recordWithArchiveIndicator = { ...record, showArchiveIndicator };
       const link = (
-        <div key={record.key ?? record[docManGuidIndex]} title={title}>
+        <div
```

Comment:
I think the title attribute may have gotten removed by accident. I think that the title attribute is for showing on narrow screens when it's too small for a table.

Reply:
The title attribute is added within `documentWithTag` function to a div element that wraps around the html content of the link variable.